graylog2-server/src/main/java/org/graylog2/bindings/MessageOutputBindings.java

```diff
@@ -48,11 +50,18 @@ public MessageOutputBindings(final Configuration configuration, ChainingClassLoa
 
     @Override
     protected void configure() {
-        final Class<? extends MessageOutput> defaultMessageOutputClass = getDefaultMessageOutputClass(BlockingBatchedESOutput.class);
+        final Class<? extends MessageOutput> defaultMessageOutputClass = getDefaultMessageOutputClass(BatchedMessageFilterOutput.class);
         LOG.debug("Using default message output class: {}", defaultMessageOutputClass.getCanonicalName());
         OptionalBinder.newOptionalBinder(binder(), Key.get(MessageOutput.class, DefaultMessageOutput.class))
                 .setDefault().to(defaultMessageOutputClass).in(Scopes.SINGLETON);
 
+        install(new OutputFilterModule());
+
+        filteredOutputsMapBinder(); // Ensure initialization of the multi-binder for filtered outputs
```

Comment:
That's our typical safety net against an uninitialized map binder if we would remove the statements below. That said, it's probably unlikely that we remove it because we wouldn't have any outputs. :smile:

So I can change it if you want. But I think it doesn't hurt.

Reply:
I don't mind keeping it in then.